commercial-schedules/src/main/java/org/dcsa/conformance/standards/cs/checks/CsDataSets.java

```diff
@@ -0,0 +1,10 @@
+package org.dcsa.conformance.standards.cs.checks;
+
+import org.dcsa.conformance.core.check.KeywordDataset;
+
+public class CsDataSets {
```

Comment:
You can 'Add a private constructor to hide the implicit public one.'
this is easy with adding the annotation: `@NoArgsConstructor(access = lombok.AccessLevel.PRIVATE)`